prow/cluster/jobs/istio/api/istio.api.all.yaml

```diff
@@ -0,0 +1,57 @@
+postsubmits:
+
+  istio/api:
+
+  - name: gen_client-go_postsubmit
+    branches:
+    - ^master$
+    - ^release-1.4$
+    decorate: true
+    path_alias: istio.io/api
+    spec:
+      containers:
+      - image: gcr.io/istio-testing/automator:0.0.1
+        command:
+        - /usr/bin/automator
+        args:
+        - --org=istio
+        - --repo=client-go
```

Comment:
Why is automator cloning the repo vs having prow do it?

Reply:
This is to decouple it from Prow. This makes it easier to run and test outside of Prow.

Reply:
If the person runs it outside of prow, can't we/shouldn't we assume they already have the repo checked out and at the commit they want?

Reply:
The purpose of the `org`, `repo`, and `branch` args to the script is so that the user does not have to make any assumptions about local filesystem state.